pyproject.toml

```diff
@@ -46,19 +46,19 @@ django-simple-history = "~2.7.0"
 djangorestframework = "==3.10.3"
 djangorestframework-gis = "==0.14"
 djangorestframework-jsonapi = "~3.0"
-djangorestframework-simplejwt = {git = "git://github.com/ShipChain/django-rest-framework-simplejwt.git", rev = "f47065b7e0cf3adf8e4095796cfc13f651a117c6"}
+djangorestframework-simplejwt = {git = "git://github.com/ShipChain/django-rest-framework-simplejwt.git", rev = "3cecdfc8050b691e51e7d45229d8bd7c03a8d6c2"}
 geocoder = "==1.38.1"
 inflection = ">=0.3.1"
 influxdb = "==5.0.0"
 logstash_formatter = "~0.5"
 mapbox = "==0.16.2"
-pyjwt = "~1.7"
 psycopg2 = ">=2.5.4"
+pyjwt = "~1.7"
 python-jose = "~3.0"
 pyyaml = ">=4.2b4"
 redis = "~3.3"
 requests = "~2.21"
-shipchain-common = "~1.0.15"
+shipchain-common = "~=1.0"
```

Comment:
I want to say that we've been intentionally pinning to the minimum patch version that we require. Theoretically, if we religiously followed semver, the `~1.0` pin would be correct. However we have breaking changes/dependences in our patch versions and we probably need the `~1.0.15` pin that was there before.